src/main/java/excel/export/controller/enricher/ResponseHeaderEnricher.java

```diff
@@ -0,0 +1,17 @@
+package excel.export.controller.enricher;
+
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletResponse;
+
+@Component
+public class ResponseHeaderEnricher {
+
+    public void enrich(HttpServletResponse response) {
```

Comment:
Cache-Control: "max-age=0"
Pragma: "public"

why?

Reply:
To be honest, that was just copy paste from one of the example.
Personally, I though that following would be enought, but my excel claimed that files were corrupted:
`response.setHeader("Content-Disposition", "attachment; filename=result.xls");
response.setContentType("application/vnd.ms-excel");`

Reply:
Ok. Thx